extensions/amp-subscriptions/0.1/amp-subscriptions.js

```diff
@@ -324,26 +324,29 @@ export class SubscriptionService {
   /**
    * Reset all platforms and re-fetch entitlements after an
    * external event (for example a login)
+   * @return {!Promise}
    */
   resetPlatforms() {
-    this.platformStore_ = this.platformStore_.resetPlatformStore();
-    this.renderer_.toggleLoading(true);
+    return this.initialize_().then(() => {
```

Comment:
I just called `initialize_` here to be safe, since it's a public method

Reply:
I'm returning the promise to help with tests. Couldn't find any places in the code that depended on `resetPlatforms` needing execute synchronously

Reply:
SGTM